README.md

```diff
@@ -22,31 +25,36 @@ installed by running this command in the terminal.
 
 `sudo apt install virtualenv git python3-dev gcc`
 
+If you are installing onto Windows, you will need to have installed Visual Studio Build Tools, and a 64-bit version of
+Python.
+
 Clone the Onko repository
 
 `git clone https://github.com/didymo/OnkoDICOM.git`
 
 Enter the directory and create a virtual environment with a name of
-your choice, in this case it's envOnkoDICOM.
+your choice, in this case it's venv.
```

Comment:
if you are going to change this, it might be best to put the virtual env outside of the cloned repo.
I didn't really know too much about what I was doing with the virtual env at the time, but in retrospect, I should have put it outside of the repo so it wouldn't show up when doing things with git.